mkdocs.yml

```diff
@@ -169,6 +176,9 @@ plugins:
           - name: gradle
             import_url: 'https://github.com/gradle/gradle?branch=master&edit_uri=/blob/master/'
             imports: [ "/*.md", "/*.png", "platforms/documentation/docs/README.md" ]
+          - name: gradle-architecture
+            import_url: 'https://github.com/gradle/gradle?branch=master&edit_uri=/blob/master/'
+            imports: [ "architecture/*", "architecture/standards/*" ]
```

Comment:
Would it make sense to import `gradle/gradle` only once? It is quite heavy

Reply:
Fixed